provision/vvv-nginx-default.conf

```diff
@@ -2,7 +2,7 @@ server {
     listen       80;
     listen       443 ssl http2;
     server_name  {vvv_hosts};
-    root         {vvv_path_to_site}/public_html;
+    root         {vvv_path_to_site}/{vvv_public_dir};
```

Comment:
Wouldn't this become a problem?

Reply:
In the context of an empty public_dir.

Reply:
you mean with no value? No idea, would need testing